lib/config/presets/bitbucket-server/index.ts

```diff
@@ -32,7 +32,7 @@ export async function fetchJSONFile(
   try {
     res = await http.getJson(url);
   } catch (err) {
-    // istanbul ignore if: not testable with nock
+    // istanbul ignore if: not testable with mock
```

Comment:
unrelated change?

Reply:
Yes, found it during debugging. Should it be done in a new PR?

Reply:
it's intentionally `nock`, so please revert

Reply:
My bad